newrelic/resource_newrelic_nrql_alert_condition.go

```diff
@@ -47,6 +47,11 @@ func resourceNewRelicNrqlAlertCondition() *schema.Resource {
 				Type:     schema.TypeBool,
 				Optional: true,
 			},
+			"violation_time_limit_seconds": {
+				Type:        schema.TypeInt,
+				Optional:    true,
+				Description: "This limit will automatically force-close a long-lasting violation after the number of hours you select",
```

Comment:
It would be great to add descriptions for all fields instead of this one being a one-off.

Reply:
@nvaller Would you mind stripping the Description field for now?  We'd like to address this with a consistent treatment that also updates the documentation.